rasa/nlu/model.py

```diff
@@ -253,9 +252,7 @@ def persist(
 
         if persistor is not None:
             persistor.persist(dir_name, model_name)
-        logger.info(
-            "Successfully saved model into '{}'".format(os.path.abspath(dir_name))
-        )
+        logger.info("Successfully saved model into '{}'".format(dir_path))
```

Comment:
Can we change that to the `f-string`? Also, I believe we'd have to use `dir_name` instead.